nightly/complete_nayduck_set.txt

```diff
@@ -104,6 +104,7 @@ pytest contracts/deploy_call_smart_contract.py
 pytest contracts/deploy_call_smart_contract.py --features nightly_protocol --features nightly_protocol_features
 pytest --timeout=240 contracts/gibberish.py
 pytest --timeout=240 contracts/gibberish.py --features nightly_protocol --features nightly_protocol_features
+pytest --timeout=400 contracts/infinite_loops.py --features nightly_protocol --features nightly_protocol_features
```

Comment:
Please also add a line without features so that it will run for the stable build as well